### PR TITLE
Add description for auto detect mode of online recovery

### PR DESCRIPTION
**Pull request by Connor1996**
Signed-off-by: Connor1996 <zbk602423539@gmail.com>

### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### What is changed, added or deleted? (Required)

Add description for auto detect mode of online recovery

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.2 (TiDB 6.2 versions)
- [x] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: 
- Other reference link(s):https://github.com/tikv/pd/pull/5403

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by rleungx — APPROVED** [review]

**Review by v01dstar — APPROVED** [review]
lgtm

File: online-unsafe-recovery.md [hunk 1]
```diff
@@ -7,11 +7,12 @@ summary: 如何使用 Online Unsafe Recovery。
...
 >
-> 此功能为有损恢复，无法保证数据和数据索引完整性。
+> - 此功能为有损恢复，无法保证数据索引一致性和事务完整性，若有问题需要额外的工具或者步骤进行相应修复。
+> - 该功能从 6.1.0 版本中引入。6.1 以下版本为实验特性，行为与该文档描述有区别，请参考相应版本文档，不推荐使用。
 
-当多数副本的永久性损坏造成部分数据不可读写时，可以使用 Online Unsafe Recovery 功能进行数据有损恢复。
+当多数副本的永久性损坏造成部分数据不可读写时，可以使用 Online Unsafe Recovery 功能进行数据有损恢复，使 TiKV 正常提供服务。
 
-## 功能说明
+## 功能说明
```
Comment (Oreoxmt) [inline online-unsafe-recovery.md:15]:
```suggestion
## 功能说明
```

File: online-unsafe-recovery.md [hunk 2]
```diff
@@ -7,11 +7,12 @@ summary: 如何使用 Online Unsafe Recovery。
 
 > **警告：**
 >
-> 此功能为有损恢复，无法保证数据和数据索引完整性。
+> - 此功能为有损恢复，无法保证数据索引一致性和事务完整性，若有问题需要额外的工具或者步骤进行相应修复。
+> - 该功能从 6.1.0 版本中引入。6.1 以下版本为实验特性，行为与该文档描述有区别，请参考相应版本文档，不推荐使用。
```
Comment (Oreoxmt) [inline online-unsafe-recovery.md:11]:
```suggestion
> - 该功能自 v6.1.0 版本开始引入。在 TiDB v6.1 以下版本为实验特性，行为与本文描述有区别，**不推荐**使用。在其他版本使用该功能时，请参考相应版本文档。
```

File: online-unsafe-recovery.md [hunk 3]
```diff
@@ -53,6 +54,8 @@ pd-ctl -u <pd_addr> unsafe remove-failed-stores <store_id1,store_id2,...>
 
 可通过 `--timeout <seconds>` 指定可允许执行恢复的最长时间。若未指定，默认为 5 分钟。当超时后，恢复中断报错。
 
+若 PD 进行过 `pd-recover` 操作，已丢失无法恢复的 TiKV 节点的 store 信息，因此无法确定要传的 store ID 时，可指定 `--auto-detect` 以允许接受一个空的 store ID 列表。在该模式下，所有未在 PD store 列表中的 store ID 均被认为无法恢复，进行移除。
```
Comment (Oreoxmt) [inline online-unsafe-recovery.md:57]:
```suggestion
若 PD 进行过灾难性恢复 [`pd-recover`](/pd-recover.md) 操作，丢失了无法恢复的 TiKV 节点的 store 信息，因此无法确定要传的 store ID 时，可指定 `--auto-detect` 参数允许传入一个空的 store ID 列表。在该模式下，所有未在 PD store 列表中的 store ID 均被认为无法恢复，进行移除。
```

**Review by Oreoxmt — COMMENTED** [review]
Rest LGTM

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: online-unsafe-recovery.md [hunk 4]
```diff
@@ -7,9 +7,10 @@ summary: 如何使用 Online Unsafe Recovery。
 
 > **警告：**
 >
-> 此功能为有损恢复，无法保证数据和数据索引完整性。
+> - 此功能为有损恢复，无法保证数据索引一致性和事务完整性，若有问题需要额外的工具或者步骤进行相应修复。
```
Comment (Oreoxmt) [inline online-unsafe-recovery.md:10]: “额外的工具或步骤”这里可以明确一下用户可以参考哪个文档进行操作吗

Comment (Connor1996) [reply online-unsafe-recovery.md:10]: 后文有写

**Pull request merged**